src/go/rpk/pkg/cli/registry/mode/mode.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package mode
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/sr"
+)
+
+func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "mode",
+		Short: "Manage schema registry mode",
+		Args:  cobra.ExactArgs(0),
```

Comment:
should this have long text? Or as parent command that's not necessary?

Reply:
Yeah, we are trying to keep it simple for parent commands, the "what is a mode" should be on our docs page, otherwise, we will dupe a lot of information here.